src/app/templates/scripts/_server.js

```diff
@@ -61,9 +61,9 @@ const tasks = {
       if (err) {
         reject(err.message);
       } else {
-        fs.writeFile(dests.css, result.css, (err) => {
-          if (err) {
-            reject(err.message);
+        fs.writeFile(dests.css, result.css, (errr) => {
```

Comment:
make one of these `error` and the other `err` rather than using `errr` as it almost seems like a miss-spelling